doc/source/dev/development_gitpod.rst

```diff
@@ -199,8 +197,8 @@ uses the rst extension with docutils.
 If you want to see the final output with the ``html`` theme you will need to 
 rebuild the docs with ``make html`` and use Live Serve as described in option 1.
 
-FAQ's
------
+FAQ's and troubleshooting
+----------------------------
```

Comment:
Are the FAQ items numbered on purpose? Maybe they could use a lower title level so they can have anchors instead of a numbered list.

Reply:
🤔  I have no preference here neither a good argument as to why titles would not be better suited. 

Will go ahead and change this. Thanks for the suggestion Matti